website/docs/docs/cloud/manage-access/scim.md

```diff
@@ -52,6 +52,11 @@ To retrieve the necessary dbt Cloud configurations for use in Okta:
     <Lightbox src="/img/docs/dbt-cloud/access-control/enable-scim.png" width="60%" title="SCIM enabled in the Okta configuration settings." />
 4. Record the **SCIM base URL** field for use in a later step.
 5. Click **Create SCIM token**.
+    :::note
+    
+    It's best practice to rotate your SCIM token at regular intervals. To do so, follow these same instructions to create a new one. Be sure not to delete the old token until it has been replaced in your IdP.
```

Comment:
```suggestion
    It's best practice to rotate your SCIM token at regular intervals. To do so, follow these same instructions to create a new one. To avoid service disruptions, don't delete the old token in dbt Cloud until you've replaced it in your IdP.
```